src/planwise/boundary/coverage.clj

```diff
@@ -8,8 +8,10 @@
     "Enumerate the supported algorithms")
 
   (compute-coverage [this point criteria]
-    "Computes a coverage area from a given geographical point"))
+    "Computes a coverage area from a given geographical point")
 
+  (as-geojson [this geometry]
+    "Converts a pg-geometry to geojson"))
```

Comment:
This is too generic to be in this particular boundary. Maybe we need a pg-util module or something similar.